doc/sphinx/Clusters_from_Scratch/active-passive.rst

```diff
@@ -27,12 +29,16 @@ Another important piece of information here is **ocf:heartbeat:IPaddr2**.
 This tells Pacemaker three things about the resource you want to add:
 
 * The first field (**ocf** in this case) is the standard to which the resource
-  script conforms and where to find it.
+  agent conforms and where to find it.
 
-* The second field (**heartbeat** in this case) is standard-specific; for OCF
-  resources, it tells the cluster which OCF namespace the resource script is in.
+* The second field (**heartbeat** in this case) the provider. This field is
+  standard-specific; for OCF resources, it tells the cluster which OCF
+  namespace the resource script is in. Some standards (e.g., **lsb**), do not
+  support this field.
```

Comment:
Actually OCF is the only one that supports it currently, so this whole paragraph could be better worded

Reply:
Done. Draft:
```
* The second field (**heartbeat** in this case) is known as the provider.
  Currently, this field is supported only for OCF resources. It tells Pacemaker
  which OCF namespace the resource script is in.
```

Reply:
Sounds good